atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolingContainer.java

```diff
@@ -68,21 +73,22 @@
 
     public CassandraClientPoolingContainer(
             MetricsManager metricsManager,
-            InetSocketAddress host,
+            CassandraServer cassandraServer,
             CassandraKeyValueServiceConfig config,
```

Comment:
cassandraServer is only used for logging. You can see that the impl of `CassandraClientFactory` has not changed. We pass in the proxy field to `CassandraClientFactory` that creates a client.